pandas/_libs/testing.pyx

```diff
@@ -89,6 +89,12 @@ cpdef assert_almost_equal(a, b,
     robj : str, default None
         Specify right object name being compared, internally used to show
         appropriate assertion message
+    index : str, default None
```

Comment:
can this be "label" or something less likely to be confused for an Index object

Reply:
I changed all the 'index' parameters to 'index_values' - I hope that is clearer.